web/storage-solution.rules

```diff
@@ -0,0 +1,18 @@
+// Returns true if the uploaded file is an image and its size is below the given number of MB.
+function isImageBelowMaxSize(maxSizeMB) {
+  return request.resource.size < maxSizeMB * 1024 * 1024
+      && request.resource.contentType.matches('image/.*');
+}
+
+service firebase.storage {
+  match /b/{bucket}/o {
+    match /{userId}/{messageId}/{fileName} {
```

Comment:
Please consolidate and migrate this into the ideal path you want (in the tests) and also update the index.js code

Reply:
Done